recommender/static/js/src/recommender.js

```diff
@@ -1,852 +1,930 @@
+        var totalNumberOfPages = Math.ceil($('.recommenderResource', element).length/ENTRIES_PER_PAGE);
+        $('.paginationItem', element).remove();
+        $('.paginationPageNumber', element).unbind();
+        if (totalNumberOfPages == 1) { return; }
+
+        /* Each paginationItem correspond to each page of resource list */
+        for (var paginationItemIndex = 1; paginationItemIndex <= totalNumberOfPages; paginationItemIndex++) {
+            var renderData = {
+                /* No previous page if current page = 1 */
+                paginationItemIndexIsOne: (paginationItemIndex == 1),
+                noMorePreviousPageIcon: (paginationItemIndex - PAGE_SPAN <= 1),
+                pageNumberIndexes: [],
+                noMoreNextPageIcon: (paginationItemIndex + PAGE_SPAN >= totalNumberOfPages),
+                /* No next page if current page is last page */
+                paginationItemIndexIsLast: (paginationItemIndex == totalNumberOfPages)
+            }
+            
+            for (var i = paginationItemIndex - PAGE_SPAN; i <= paginationItemIndex + PAGE_SPAN; i++) {
+                renderData.pageNumberIndexes.push({
+                    pageNumberIndex: i,
+                    pageNumberIndexIsActive: (i == paginationItemIndex),
+                    pageNumberIndexOutOfRange: (i <= 0 || i > totalNumberOfPages)
+                });
+            }
+
+            var paginationItemDiv = $(Mustache.render($("#paginationItemTemplate").html(), renderData));
+            $('.pagination', element).append(paginationItemDiv);
+        }
+
+        /* Page-changing event */
+        $('.paginationPageNumber', element).click(function () {
+            var previousPage = CURRENT_PAGE.toString();
+            if ($(this).hasClass('morePageIcon')) {
+                Logger.log('mit.recommender.pagination', generateLog(loggerStatus['pagination']['moreIcon']));
+                return;
+            }
+            else if ($(this).hasClass('previousPageIcon')) {
+                CURRENT_PAGE -= 1;
+            }
+            else if ($(this).hasClass('nextPageIcon')) { CURRENT_PAGE += 1; }
+            else { CURRENT_PAGE = parseInt($(this).text()); }
+            var status = loggerStatus['pagination']['toPageNIcon'](previousPage, CURRENT_PAGE.toString())
+            Logger.log('mit.recommender.pagination', generateLog(status));
+            pagination();
+        });
+    }
+
+    /**
+     * Export all resources from the Recommender. This is intentionally not limited to staff
+     * members (community contributions do not belong to the course staff). Sensitive
+     * information is exported *is* limited (flagged resources, and in the future, PII if
+     * any).
+     */
+    function bindExportResourceEvent() {
+        $.ajax({
+            type: "POST",
+            url: exportResourceUrl,
+            data: JSON.stringify({}),
+            success: function(result) {
+                if (result['Success'] == true) {
```

Comment:
Your `success` AJAX handler shouldn't need to check if the response succeeded -- it already knows, based on the HTTP status code.